sale_manual_delivery/models/stock_move.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2017 Denis Leemann, Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    carrier_id = fields.Many2one("delivery.carrier", string="Delivery Method")
```

Comment:
I think it is better to have the carrier on stock.picking level, I don't know cases when moves belong to one picking and served by different carriers

Reply:
Hi @i-vyshnevska
Agree with you. However, this module is migrated from https://github.com/OCA/sale-workflow/pull/987. I think it is better to have this reviewed in mentioned PR first. I'll notify this and tag you to follow up